cli.go

```diff
@@ -71,6 +72,7 @@ func (cli *CLI) Run(args []string) int {
 	flags.StringVar(&awsAccessKey, "aws-access-key", "", "AWS access key used in deep check mode.")
 	flags.StringVar(&awsSecretKey, "aws-secret-key", "", "AWS secret key used in deep check mode.")
 	flags.StringVar(&awsRegion, "aws-region", "", "AWS region used in deep check mode.")
+	flags.BoolVar(&exitErrorStatus, "exit-error-status", false, "return exit status code when issue exists.")
```

Comment:
I'm sorry... Please use another option name 🙇 
I think `--exit-error-status` is bad name. (My recommendation is `--error-with-issues`)